tests/adapter_tests/aws/test_aws_lambda.py

```diff
@@ -318,3 +318,41 @@ def test_oauth(self):
         assert response["statusCode"] == 200
         assert response["headers"]["content-type"] == "text/html; charset=utf-8"
         assert "https://slack.com/oauth/v2/authorize?state=" in response.get("body")
+
+    @mock_lambda
+    def test_oauth_redirect(self):
+        app = App(
```

Comment:
If we have a mock state_store, which returns a fixed value "uuid4-value", the test can be better. Can you check this?

Reply:
@tattee Thanks for asking this! I thought that we need to implement state_store with `issue`/`consume` methods (returning a fixed value "uuid4-value" means having the issue method). But, for this test, having only `consume` method is required. Thus, the following diff should work for you!

```diff
diff --git a/tests/adapter_tests/aws/test_aws_lambda.py b/tests/adapter_tests/aws/test_aws_lambda.py
index b92300e..1b0166c 100644
--- a/tests/adapter_tests/aws/test_aws_lambda.py
+++ b/tests/adapter_tests/aws/test_aws_lambda.py
@@ -3,6 +3,7 @@ from time import time
 from urllib.parse import quote
 
 from moto import mock_lambda
+from slack_sdk.oauth import OAuthStateStore
 from slack_sdk.signature import SignatureVerifier
 from slack_sdk.web import WebClient
 
@@ -321,6 +322,10 @@ class TestAWSLambda:
 
     @mock_lambda
     def test_oauth_redirect(self):
+        class TestStateStore(OAuthStateStore):
+            def consume(self, state: str) -> bool:
+                return state == "uuid4-value"
+
         app = App(
             client=self.web_client,
             signing_secret=self.signing_secret,
@@ -328,6 +333,7 @@ class TestAWSLambda:
                 client_id="111.111",
                 client_secret="xxx",
                 scopes=["chat:write", "commands"],
+                state_store=TestStateStore(),
             ),
         )
 
@@ -340,7 +346,7 @@ class TestAWSLambda:
             "isBase64Encoded": False,
         }
         response = SlackRequestHandler(app).handle(event, self.context)
-        assert response["statusCode"] == 401
+        assert response["statusCode"] == 200
         assert response["headers"]["content-type"] == "text/html; charset=utf-8"
         assert response.get("body") is not None
 
@@ -353,6 +359,6 @@ class TestAWSLambda:
             "isBase64Encoded": False,
         }
         response = SlackRequestHandler(app).handle(event, self.context)
-        assert response["statusCode"] == 401
+        assert response["statusCode"] == 200
         assert response["headers"]["content-type"] == "text/html; charset=utf-8"
         assert response.get("body") is not None
```